src/utils.ts

```diff
@@ -19,9 +19,13 @@ export function checkFileChanges(context: ExtensionContext): void {
 }
 
 export function createUserId(context: ExtensionContext) {
-  if (!context.workspaceState.get('userId')) {
-    context.workspaceState.update('userId', uniqid());
+  let userId = context.workspaceState.get('userId') as string | undefined;
```

Comment:
why the casting ?

Reply:
default type for context.workspaceState.get('userId')  is {} | undefined, but it's a string, not an object 😮